components/radio/stories/radio.stories.js

```diff
@@ -91,31 +95,132 @@ export default {
 	},
 };
 
-export const Default = (args) => html`
+// ===== Story specific templates ======
+
+// @todo Radio needs more robust coverage of all states, including hover, focus, etc.
+const ChromaticGroupTemplate = (args) => html`
 	<div style="display: flex; flex-direction: column; align-items: flex-start;">
 		${Template({
 			...args,
-			label: "Default"
+			label: "Default, not selected",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-1",
 		})}
 		${Template({
 			...args,
-			isEmphasized: true,
 			isChecked: true,
-			label: "Emphasized radio button label that is so long it has to wrap",
+			isEmphasized: false,
+			label: "Default, selected",
 			customStyles: {
 				"max-width": "220px",
-			}
+			},
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-2",
+		})}
+		${Template({
+			...args,
+			isChecked: true,
+			isEmphasized: true,
+			label: "Emphasized, selected",
+			customStyles: {
+				"max-width": "220px",
+			},
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-3",
 		})}
 		${Template({
 			...args,
 			isDisabled: true,
-			label: "Disabled"
+			label: "Disabled",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-4",
 		})}
 		${Template({
 			...args,
 			isDisabled: true,
 			isReadOnly: true,
-			label: "Read only"
+			label: "Read-only",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-5",
 		})}
 	</div>
 `;
+
+const BasicGroupTemplate = (args) => html`
+	<div style="display: flex; flex-direction: column; align-items: flex-start;">
+		${Template({
+			...args,
+			label: "Example label",
+			id: "radio-1-" + args?.id ?? "default",
+			name: "radio-example-" + args?.name ?? "default",
+		})}
+		${Template({
+			...args,
+			isChecked: true,
+			label: "Inititally selected radio button that has wrapping label text",
+			customStyles: {
+				"max-width": "220px",
+			},
+			id: "radio-2-" + args?.id ?? "default",
+			name: "radio-example-" + args?.name ?? "default",
+		})}
+	</div>
+`;
+
+// ===== Stories ======
+
+export const Default = (args) => {
+	return window.isChromatic() ? ChromaticGroupTemplate(args) : BasicGroupTemplate(args);
+};
+Default.tags = ["!autodocs"];
```

Comment:
Aaahhh ok I see what you did here. This default story is the one with chromatic attached, but then you've removed it from the autodocs while keeping it in the sidenav. 

Then for the "Standard" story (which is the same as the Default when `isChromatic` is false), you removed it from the sidenav. 

This might be how we get around some of the docs exceptions you were trying to figure out in 2885 right? 🤔